setup.py

```diff
@@ -25,6 +25,9 @@
       # Include the separately-compiled shared library
       author="Continuum Analytics, Inc.",
       packages=packages,
+      package_data={
+        'pygdf.tests': ['data/*.pickle'],
+      },
```

Comment:
In the future we should look into removing these data files and generate data for the tests affected programmatically. Not a blocker for now though.